packages/web-components/fast-components/src/radio-group/radio-group.template.ts

```diff
@@ -0,0 +1,15 @@
+import { html, slotted } from "@microsoft/fast-element";
+import { RadioGroup } from "./radio-group";
+
+export const RadioGroupTemplate = html<RadioGroup>`
+    <template
+        role="radiogroup"
+        aria-disabled="${x => x.disabled}"
+        aria-readonly="${x => x.readOnly}"
+    >
+        <slot name="label"></slot>
+        <div class="control" part="control">
```

Comment:
"control" seems like an awkward name for the region containing the controls. Perhaps this should be "group" or "region" or "control-region" or something else to describe this as a container element?

Reply:
we used positioning-region for slider and I think another control, so I'll use that for consistency sake unless there are objections